cmd/queue/main.go

```diff
@@ -155,9 +155,21 @@ func proxyHandler(reqChan chan queue.ReqEvent, breaker *queue.Breaker, tracingEn
 
 		// Enforce queuing and concurrency limits.
 		if breaker != nil {
+			var cf func()
+			if tracingEnabled {
```

Comment:
Is this the right way to do it?
I though the instrumentation was supposed to be always there and the sampling/enabling is done at the exporter level.
Maybe not for hot path methods...

Reply:
I think that's why they are out. But this code predates me :)

Reply:
Sounds like it'd be good cleanup to try in early 0.15?

Reply:
I'll add an issue to look into this.

Reply:
See https://github.com/knative/serving/pull/6538 for background on this. Every span is 3 allocations essentially, we try to keep allocations on hot-paths to an absolute minimum.